MonoGame.Framework/Curve.cs

```diff
@@ -3,81 +3,106 @@
 // file 'LICENSE.txt', which is part of this source code package.
 
 using System;
+using System.ComponentModel;
 using System.Runtime.Serialization;
 
 namespace Microsoft.Xna.Framework
 {
-	// TODO [TypeConverter(ExpandableObjectConverter)]
+    /// <summary>
+    /// Contains a collection of <see cref="CurveKey"/> points in 2D space and provides methods for evaluating features of the curve they define.
+    /// </summary>
+    // TODO : [TypeConverter(typeof(ExpandableObjectConverter))]
     [DataContract]
     public class Curve
     {
         #region Private Fields
 
-        private CurveKeyCollection keys;
-        private CurveLoopType postLoop;
-        private CurveLoopType preLoop;
-
-        #endregion Private Fields
+        private CurveKeyCollection _keys;
+        private CurveLoopType _postLoop;
+        private CurveLoopType _preLoop;
```

Comment:
I tried avoid those fixes, but sometimes hand direct me to do this. In this PR however there is no significant formatting fixes.